src/redux/pattern/pattern.sagas.js

```diff
@@ -0,0 +1,122 @@
+import { takeEvery, call, put } from 'redux-saga/effects';
+import { push } from 'connected-react-router';
+import {
+  setPatterns,
+  setPatternLoading,
+  setPattern,
+  setPatternError
+} from './pattern.actions';
+
+import {
+  getAllPatterns,
+  deletePattern,
+  createPattern,
+  updatePattern,
+  getPatternById
+} from './pattern.operations';
+
+import { config, routes } from '../../configs';
+
+import {
+  setSnackBarSeverity,
+  setSnackBarStatus,
+  setSnackBarMessage
+} from '../snackbar/snackbar.actions';
+import {
+  GET_PATTERNS,
+  DELETE_PATTERN,
+  ADD_PATTERN,
+  UPDATE_PATTERN,
+  GET_PATTERN
+} from './pattern.types';
+
+const {
+  SUCCESS_ADD_STATUS,
+  SUCCESS_DELETE_STATUS,
+  SUCCESS_UPDATE_STATUS
+} = config.statuses;
+
+function* handlePatternsLoad() {
+  try {
+    yield put(setPatternLoading(true));
+    const patterns = yield call(getAllPatterns, null);
+    yield put(setPatterns(patterns));
+    yield put(setPatternLoading(false));
+  } catch (error) {
+    yield call(handlePatternError, error);
+  }
+}
+
+function* handlePatternLoad({ payload }) {
+  try {
+    yield put(setPatternLoading(true));
+    const pattern = yield call(getPatternById, payload);
+    console.log(pattern);
```

Comment:
Please, remove the useless `console.log`

Reply:
fixed